osquery/events/CMakeLists.txt

```diff
@@ -111,6 +112,7 @@ function(generateOsqueryEvents)
       windows/usn_journal_reader.h
       windows/windowseventlogpublisher.h
       windows/windowseventlogparserservice.h
+      windows/windowseventlogutils.h
```

Comment:
A better name for these files could be 'windowseventlogparser.h/cpp' since it better matches what's inside